src/style.css

```diff
@@ -1,3 +1,88 @@
-body {
+
+* {
   margin: 0;
+  box-sizing: border-box;
+
+  --active-link-color: #00ACDC;
+
+}
+
+a {
+  color: #000;
+  text-decoration: none;
+  height: 60px;
+  display: flex;
+  align-items: center;
+
+
+}
+
+img {
+  height: 40px;
+  width: 40px;
+  align-self: center;
+
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+
+header {
+  display: flex;
+  justify-content: space-between;
+  margin: auto;
+  padding: 0 50px;
+}
+
+nav {
+  display: flex;
+
+
+}
+
+ul {
+  padding: 0;
+  align-items: center;
+  display: flex;
+
+}
+
+li {
+  text-transform: uppercase;
+  list-style: none;
+  margin-right: 20px;
+  height: 100%;
+  display: flex;
+  align-items: center;
+
+}
+
+li:last-child {
+  margin-right: 0;
+}
+
+a:hover {
```

Comment:
Don't use tags for styling, use classes instead

Reply:
@nazarmatsevych all tags replaced with classes